views/install.sh.erb

```diff
@@ -26,6 +26,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+download_timeout=10
```

Comment:
the problem here is that this most likely needs to be a lot higher.  people may be in 1.5Mbps ADSL links in australia trying to download from omnitruck.  it most certainly needs to be tweakable.  looks like the semantics of this may also different depending on the application.